File: cpp/Processor.cpp
#include <iostream>
#include <string>

using namespace std;

class Processor
{
private:
    string NameProcessor;
    int PriceProcessor;

public:
    Processor(string Name, int Price)
    {
        this->NameProcessor = Name;
        this->PriceProcessor = Price;
    }

    Processor() {}

    void setNameProcessor(string Name)
    {
        this->NameProcessor = Name;
    }

    string getNameProcessor()
    {
        return this->NameProcessor;
    }

    void setPriceProcessor(int Price)
    {
        this->PriceProcessor = Price;
    }

    int getPriceProcessor()
    {
        return this->PriceProcessor;
    }

    void print()
    {
        cout << "Processor :" << endl;
        cout << endl;
        cout << "Name -> " << this->getNameProcessor() << endl;
        cout << "Price (Rp) ->" << this->getPriceProcessor() << endl;
    }

    ~Processor() {}
};

File: python/Processor.py
class Processor:
    __NameProcessor = ""
    __PriceProcessor = 0

    def __init__(self, Name="", Price=0):
        self.__NameProcessor = Name
        self.__PriceProcessor = Price

    def setNameProcessor(self, Name):
        self.__NameProcessor = Name

    def getNameProcessor(self):
        return self.__NameProcessor

    def setPriceProcessor(self, Price):
        self.__PriceProcessor = Price

    def getPriceProcessor(self):
        return self.__PriceProcessor

    def print(self):
        print("Processor:")
        print("Name : " + str(self.getNameProcessor()))
        print("Price (Rp) :" + str(self.getPriceProcessor()))